### ISSUE TITLE
Description webview loads slowly

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode-pull-request-github#1037]
I've been seeing this when loading the description page for 1-2 seconds:

![image](https://user-images.githubusercontent.com/2193314/54052345-01f2c580-4199-11e9-9fd9-a2d78058f715.png)

I suspect there is a problem with how the page is laid out (or how things are scripted) which is preventing rendering the work in progress layout before the entire page is loaded.

**Comment by RMacfarlane** [comment]
Yup, currently the structure is that we fetch a bunch of information from GitHub, and then dispatch just one initialization message to the webview
